tests/api/test_main.py

```diff
@@ -344,59 +344,37 @@ def selected_values_test():
     selected_values_test()
 
 
-# combined test for read_politician_search and read_politicians_image_scanner
-def test_search_and_image_scanner():
-    search_response = client.get("/search?text=Philipp")
-    image_scanner_response = client.get("image-scanner?text=Philipp")
-
-    assert search_response.json() == image_scanner_response.json()
-
-
```

Comment:
If the /politician/{id}/votes route is not supposed to be deleted, please "add the test_read_politician_votes()" test back.

You do not need to add the "test_search_and_image_scanner()" test. It is too slow, and not that useful, so I deleted it.